### ISSUE TITLE
remote cracking for hashes with hashcat like autoresp does

### ISSUE THREAD

**Issue by thesle3p** [DanMcInerney/icebreaker#8]
Would it be possible to have icebreaker send hashes to be cracked on a remote server over ssh as opposed to locally by JTR?

**Comment by DanMcInerney** [comment]
Yes it would. That's gonna take a pretty solid amount of effort to implement so I won't put a timetable on it but I'll try to implement this at some point in the near future.
